src/test/resources/conf/local.conf.json

```diff
@@ -4,7 +4,7 @@
   "key": "BROWSERSTACK_ACCESS_KEY",
 
   "capabilities": {
-    "build": "cucumber-java-browserstack",
+    "build": "browserstack-build-1",
```

Comment:
Can we add session name?

Reply:
sessionName automatically gets picked up as per scenario like we do with SDK. 
- Comment from @kamal-kaur04